### ISSUE TITLE
[Bug]: metadata is not displayed if '#' (hash char) is in wikilink alias

### ISSUE THREAD

**Issue by jannismoeller** [mgmeyers/obsidian-kanban#456]
### Describe the bug

When using the hash character in the link alias of a wikilink but not in the path, a note's metadata is not displayed:

![MetadataHashBug](https://user-images.githubusercontent.com/30196967/152153783-e988e7f7-a0d2-476c-bdbe-a82a7fd35050.png)

### Expected Behavior

Metadata should be displayed the same as for an alias without '#'.

### Steps to reproduce

Obsidian version: v0.13.23
Obsidian-Kanban version: v1.2.28

[TestKanban_v1.2.28.md](https://github.com/mgmeyers/obsidian-kanban/files/7986372/TestKanban_v1.2.28.md):

    ---
    kanban-plugin: basic
    ---
    
    ## TestList
    
    - [ ] [[TestCard|Hash # and Pipe | in alias]] 
    
    
    %% kanban:settings
    ```
    {"kanban-plugin":"basic","metadata-keys":[{"metadataKey":"metadata","label":"","shouldHideLabel":false,"containsMarkdown":false}]}
    ```
    %%

[TestCard.md](https://github.com/mgmeyers/obsidian-kanban/files/7986373/TestCard.md)

    ---
    metadata: Data
    ---

### Operating system

Windows

**Comment by jannismoeller** [comment]
The problem is the parsing of the wikilink in this function:

https://github.com/mgmeyers/obsidian-kanban/blob/3849e700668cbd831ed0a38432cee2d1b4f66aef/src/helpers/renderMarkdown.ts#L25

for the example `[[TestCard|Hash # and Pipe | in alias]]` it returns the following normalized path:

    {root: "TestCard|Hash ", subpath: "# and Pipe ", alias: " in alias"}

while it should be:

    {root: "TestCard", subpath: "", alias: "Hash # and Pipe | in alias"}

with the incorrect root, it is not able to find the file with the metadata.

**Comment by jannismoeller** [comment]
Fixed in c26d7404d14520d33eda47805bb6dba3cc6584f3
